index.html

```diff
@@ -14,12 +14,12 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
        
         <div class="name">
             <label for="name" id="name-label">Name</label>
-            <input type="text" id="name" name="name" placeholder="Enter your name" required>
+            <input type="text" id="name" name="name" placeholder="Name" required>
         </div>
 
         <div class="email">
             <label for="email" id="email-label">Email</label>
-            <input type="email" name="email" id="email" placeholder="Enter your Email" required>
+            <input type="email" name="email" id="email" placeholder="Email" required>
         </div>
         
```

Comment:
the input file should wrapped in the form to have more information check this: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form